react-file-uploads/server/server.ts

```diff
@@ -0,0 +1,77 @@
+import 'dotenv/config';
+import express from 'express';
+import pg from 'pg';
+import {
+  ClientError,
+  errorMiddleware,
+  uploadsMiddleware,
+} from './lib/index.js';
+
+type Image = {
+  imageId: number;
+  url: string;
+  caption: string;
+};
+
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.use(express.static('public'));
+app.use(express.json());
+
+app.post(
+  '/api/uploads',
+  uploadsMiddleware.single('image'),
+  async (req, res, next) => {
+    try {
+      if (!req.file) throw new ClientError(400, 'no file field in request');
+      const { caption } = req.body as Partial<Image>;
+      if (!caption) {
+        throw new ClientError(400, 'caption is a required field');
+      }
+      /* TODO:
+       * - create a url for the image by combining '/images/' with req.file.filename
+       * - insert the "caption" and "url" into the "images" table
+       * - respond with the inserted row data and the proper status code for inserting data
+       */
+
+      const url = `/image/${req.file.filename}`;
```

Comment:
This should probably be `/images/` instead of `/image/` (plural). However, I can't tell for certain because I don't see a `/server/public/images` folder in your repo. Did you create that folder? Are the images being stored there when you upload?